src/Css/Gradients.elm

```diff
@@ -0,0 +1,1287 @@
+underTheLake =
+    gradient2 "093028" "237A57"
+
+
+theBlueLagoon : Style
+theBlueLagoon =
+    gradient2 "43C6AC" "191654"
+
+
+canYouFeelTheLoveTonight : Style
+canYouFeelTheLoveTonight =
+    gradient2 "4568DC" "B06AB3"
+
+
+veryBlue : Style
+veryBlue =
+    gradient2 "0575E6" "021B79"
+
+
+loveAndLiberty : Style
+loveAndLiberty =
+    gradient2 "200122" "6f0000"
+
+
+orca : Style
+orca =
+    gradient2 "44A08D" "093637"
+
+
+venice : Style
+venice =
+    gradient2 "6190E8" "A7BFE8"
+
+
+pacificDream : Style
+pacificDream =
+    gradient2 "34e89e" "0f3443"
+
+
+learningAndLeading : Style
+learningAndLeading =
+    gradient2 "F7971E" "FFD200"
+
+
+celestial : Style
+celestial =
+    gradient2 "C33764" "1D2671"
+
+
+purplepine : Style
+purplepine =
+    gradient2 "20002c" "cbb4d4"
+
+
+shaLaLa : Style
+shaLaLa =
+    gradient2 "D66D75" "E29587"
+
+
+mini : Style
+mini =
+    gradient2 "30E8BF" "FF8235"
+
+
+maldives : Style
+maldives =
+    gradient2 "B2FEFA" "0ED2F7"
+
+
+cinnamint : Style
+cinnamint =
+    gradient2 "4AC29A" "BDFFF3"
+
+
+html : Style
+html =
+    gradient2 "E44D26" "F16529"
+
+
+coal : Style
+coal =
+    gradient2 "EB5757" "000000"
+
+
+sunkist : Style
+sunkist =
+    gradient2 "F2994A" "F2C94C"
+
+
+blueSkies : Style
+blueSkies =
+    gradient2 "56CCF2" "2F80ED"
+
+
+chittyChittyBangBang : Style
+chittyChittyBangBang =
+    gradient2 "007991" "78ffd6"
+
+
+visionsOfGrandeur : Style
+visionsOfGrandeur =
+    gradient2 "000046" "1CB5E0"
+
+
+crystalClear : Style
+crystalClear =
+    gradient2 "159957" "155799"
+
+
+mello : Style
+mello =
+    gradient2 "c0392b" "8e44ad"
+
+
+compareNow : Style
+compareNow =
+    gradient2 "EF3B36" "FFFFFF"
+
+
+meridian : Style
+meridian =
+    gradient2 "283c86" "45a247"
+
+
+relay : Style
+relay =
+    gradient3 "3A1C71" "D76D77" "FFAF7B"
+
+
+alive : Style
+alive =
+    gradient2 "CB356B" "BD3F32"
+
+
+scooter : Style
+scooter =
+    gradient2 "36D1DC" "5B86E5"
+
+
+terminal : Style
+terminal =
+    gradient2 "000000" "0f9b0f"
+
+
+telegram : Style
+telegram =
+    gradient2 "1c92d2" "f2fcfe"
+
+
+crimsonTide : Style
+crimsonTide =
+    gradient2 "642B73" "C6426E"
+
+
+socialive : Style
+socialive =
+    gradient2 "06beb6" "48b1bf"
+
+
+subu : Style
+subu =
+    gradient3 "0cebeb" "20e3b2" "29ffc6"
+
+
+shift : Style
+shift =
+    gradient3 "000000" "E5008D" "FF070B"
+
+
+clot : Style
+clot =
+    gradient3 "070000" "4C0001" "070000"
+
+
+brokenHearts : Style
+brokenHearts =
+    gradient2 "d9a7c7" "fffcdc"
+
+
+kimobyIsTheNewBlue : Style
+kimobyIsTheNewBlue =
+    gradient2 "396afc" "2948ff"
+
+
+dull : Style
+dull =
+    gradient2 "C9D6FF" "E2E2E2"
+
+
+purpink : Style
+purpink =
+    gradient2 "7F00FF" "E100FF"
+
+
+orangeCoral : Style
+orangeCoral =
+    gradient2 "ff9966" "ff5e62"
+
+
+summer : Style
+summer =
+    gradient2 "22c1c3" "fdbb2d"
+
+
+kingYna : Style
+kingYna =
+    gradient3 "1a2a6c" "b21f1f" "fdbb2d"
+
+
+velvetSun : Style
+velvetSun =
+    gradient2 "e1eec3" "f05053"
```

Comment:
This is only interpreted when elm-css run, no useless gradients are added on index.css